plugins/lang/python/python_loader.cc

```diff
@@ -443,9 +443,13 @@ runPythonScript(std::shared_ptr<PythonContext> titl,
     // if we're simply executing the body of the script
     try {
         if(elementToRun == PackageElement::MAIN) {
-                JML_TRACE_EXCEPTIONS(false);
-                boost::python::object obj
-                    = boost::python::exec(scriptSourceStr, pyControl.main_namespace);
+            JML_TRACE_EXCEPTIONS(false);
+            char argv1[] = "mldb-boost-python";
+            char *argv[] = {argv1};
+            int argc = sizeof(argv[0]) / sizeof(char *);
+            PySys_SetArgv(argc, argv);
```

Comment:
Would we need the same PySys_SetArgv call for the python::exec call at line 471?